sdk/spring/azure-spring-integration-eventhubs/src/test/java/com/azure/spring/integration/eventhub/converter/EventHubMessageConverterTest.java

```diff
@@ -5,31 +5,38 @@
 
 import com.azure.messaging.eventhubs.EventData;
 import com.azure.spring.integration.core.AzureHeaders;
+import com.azure.spring.integration.core.EventHubHeaders;
 import com.azure.spring.integration.core.converter.AzureMessageConverter;
 import com.azure.spring.integration.test.support.AzureMessageConverterTest;
 import org.junit.Test;
 import org.springframework.messaging.MessageHeaders;
 import org.springframework.messaging.support.NativeMessageHeaderAccessor;
 import org.springframework.util.LinkedMultiValueMap;
 
+import java.time.Instant;
+import java.time.temporal.ChronoUnit;
 import java.util.Arrays;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 
 import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertSame;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
 
 public class EventHubMessageConverterTest extends AzureMessageConverterTest<EventData> {
 
     private static final String EVENT_DATA = "event-hub-test-string";
-    private static final int PARTITION_ID = 1;
+    private static final String RAW_PARTITION_ID = "1";
```

Comment:
It should be PARTITION_ID here, the library has two fields related to partition id,

- PARTITION_ID
- RAW_PARTITION_ID
the PARTITION_ID is used when users want to set partition id sending the message,
the RAW_PARTITION_ID is used when receiving message from the brokers.